src/category_theory/discrete_category.lean

```diff
@@ -34,6 +34,12 @@ by { dsimp [discrete], apply_instance }
 
 variables {C : Type u₂} [category.{v₂} C]
 
+instance {I : Type u₁} {i j : discrete I} (f : i ⟶ j) : is_iso f :=
+{ inv := ⟨⟨f.1.1.symm⟩⟩, }
+
+def iso {I : Type u₁} {i j : discrete I} (h : i = j) : i ≅ j :=
+{ hom := ⟨⟨h⟩⟩, inv := ⟨⟨h.symm⟩⟩, }
```

Comment:
Isn't this just `eq_to_iso`?

Reply:
For some reason I had thought it was different, but now can't explain why. I'd wanted it on some other branch, so I'll sort it out when I return to that.